### PR TITLE
added functionalities with mockbee

### PR DESCRIPTION
**Pull request by anasm11**
To check:
/src/context
/src/components
  -Cart
  -Wishlist
  -Login/signup
  -Products

### CODE REVIEW

File: src/components/Cart.jsx [hunk 1]
```diff
@@ -1,8 +1,13 @@
-import React from 'react'
+import axios from 'axios'
+import React, { useState, useEffect } from 'react'
 import { productsData, IcRoundFavoriteBorder, IcRoundShare, IcSharpMoreVert } from '../assets/index'
 import '../css/Cart.css'
+import { useCart } from "../context/CartContext"
 
 const Cart = () => {
+  const { cartItems, removeFromCart, incrementCartItem, decrementCartItem } = useCart()
+  console.log(cartItems, 'cartitemsusecntct')
```
Comment (Naman-Saxena1) [inline src/components/Cart.jsx:9]: Can remove commented code

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks fine 👍 
Was not able to review UI because of Netlify error

File: src/context/CartContext.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  const removeFromCart = async (product) => {
+    const res = await axios({
+      method: "delete",
+      url: `/api/user/cart/${product._id}`,
+      headers: {
+        authorization: localStorage.getItem('token')
+      }
+
+    })
+    console.log(res.data)
```
Comment (Sahil25061999) [inline src/context/CartContext.js:43]: remove console.log if not needed

File: src/components/Login.jsx [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
+                <span className='single-line'>
+                    <div className='checkbox-input'>
+                        <input className='checkbox' type='checkbox' name='remember'/>
+                        <label >Remember me</label>
+                    </div>
+                    <div className='link'>Forgot password?</div>
+                </span>
+                <button className='btn primary-btn' onClick={async()=>{
+                    const res=await axios.post('/api/auth/login',{email,password})
+                    console.log(res)
```
Comment (Sahil25061999) [inline src/components/Login.jsx:25]: from here too

**Review by Sahil25061999 — APPROVED** [review]
code looks good to me, require some minor changes

**Pull request merged**